x/ecocredit/msgs.go

```diff
@@ -12,6 +12,17 @@ var (
 		&MsgRetire{}, &MsgCancel{}
```

Comment:
```suggestion
		&MsgRetire{}, &MsgCancel{}
		_, _, _, _, _ legacytx.LegacyMsg = &MsgCreateClass{}, &MsgCreateBatch{}, &MsgSend{},
		&MsgRetire{}, &MsgCancel{}
```

not strictly needed, but this makes sure we didn't make typos in function signatures. Same for x/group